Client.UnitTests/TestDataProvider.cs

```diff
@@ -33,6 +33,21 @@ public static IEnumerable<TestCaseData> Provider()
                     }, new ActivateJobsResponse(),
                     (RequestCreator<IActivateJobsResponse>)
                     (zeebeClient => zeebeClient.NewActivateJobsCommand().JobType("type").MaxJobsToActivate(12)));
+                yield return new TestCaseData(
+                    new TopologyRequest
+                    {
+                        ProcessInstanceKey = 12113
```

Comment:
Sorry my example was bit wrong :see_no_evil: here the key makes no sense, I'm wondering that this compiles :laughing: 

But it is fine I can fix it later :) I will merge it anyway Thanks @askalik for all your efforts :rocket: